tests/test_hear_keyboard_interrupt.py

```
@@ -58,37 +66,84 @@
+	Expected exit code when process receives SIGINT.
+
+	`130` = `128` + `SIGINT(2)` as per POSIX convention.
+	"""
+
+	INVALID_ARGS_EXIT_CODE: int = 2
+	"""Exit code indicating invalid command-line arguments."""
+
+	TEST_MULTICAST_GROUP: str = "224.0.0.1"
+	"""Standard multicast group address for testing."""
+
+	COVERAGE_CMD_TEMPLATE: str = f"{str(sys.executable)} -m coverage run -p --context=Integration"
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)